packages/vuetify/src/effects/elevation/index.ts

```diff
@@ -0,0 +1,22 @@
+// Setup
+import { computed } from 'vue'
+
+// Types
+interface ElevationProps {
```

Comment:
I know this is on me, but do we maybe want to change name of interface to not be the same as the function? I don't know how people feel about prefix `I`?

Reply:
what function?

Reply:
https://github.com/vuetifyjs/vuetify/pull/10315/files#diff-5669027a74cdf9363f9f184608be74e9R10

Reply:
Ah yeah, missed that
Alternative is to change the function name, i.e. `ElevationPropsFactory`

Reply:
Also we're using camelCase for functions, here it's PascalCase. So `elevationProps` function would solve the issue as well, but since it's actually a factory then the name should reflect that (maybe `createElevationProps` would be better)

Reply:
True. I think I had some thought to make it Pascal but now I don't rememeber

Reply:
I believe from our conversation majority sided with `makeElevationProps` due to it being 1 letter less than `createElevationProps`. Any final feedback before this is implemented?

Reply:
2 less. I am fine with make, but as noted would also be fine with just `elevationProps()`

Reply:
I'm good with either as well. I like elevationProps as it is more simple and I like makeElevationProps because it is descriptive. Overall I say I'd lean towards elevationProps.